Sources/Extensions/CALayer+Installer.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+public extension CALayer {
+
+    /// Apply new configuration state without frame updating.
+    ///
+    /// - note: Use `DEFAULT_STATE` for setting the state to the default value.
+
+    var nx_state: AnyHashable {
```

Comment:
Actually, we don't use states, so maybe we should deprecate it.

Reply:
What is the reason of deprecation for message?

Reply:
I'm not sure what you mean... Ping me in Slack DMs